testproject/Assets/Tests/Manual/Scripts/NetworkManagerHud.cs

```diff
@@ -0,0 +1,40 @@
+using MLAPI;
```

Comment:
I would recommend this superior NetworkManagerHud 😃
https://github.com/Unity-Technologies/mlapi-community-contributions/blob/main/com.mlapi.contrib.extensions/Runtime/NetworkManagerHud/NetworkManagerHud.cs

Reply:
what a shameless plug that was! :)